README.md

```diff
@@ -32,46 +32,48 @@ if (ci.isCI) {
-| [Woodpecker](https://woodpecker-ci.org/)                                        | `ci.WOODPECKER`      | ✅   |
+| Name                                                                            | Constant                | isPR |
+| ------------------------------------------------------------------------------- | --------------------    | ---- |
+| [AWS CodeBuild](https://aws.amazon.com/codebuild/)                              | `ci.CODEBUILD`          | 🚫   |
+| [AppVeyor](http://www.appveyor.com)                                             | `ci.APPVEYOR`           | ✅   |
+| [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) | `ci.AZURE_PIPELINES`    | ✅   |
+| [Appcircle](https://appcircle.io/)                                              | `ci.APPCIRCLE`          | 🚫   |
+| [Bamboo](https://www.atlassian.com/software/bamboo) by Atlassian                | `ci.BAMBOO`             | 🚫   |
+| [Bitbucket Pipelines](https://bitbucket.org/product/features/pipelines)         | `ci.BITBUCKET`          | ✅   |
+| [Bitrise](https://www.bitrise.io/)                                              | `ci.BITRISE`            | ✅   |
+| [Buddy](https://buddy.works/)                                                   | `ci.BUDDY`              | ✅   |
+| [Buildkite](https://buildkite.com)                                              | `ci.BUILDKITE`          | ✅   |
+| [CircleCI](http://circleci.com)                                                 | `ci.CIRCLE`             | ✅   |
+| [Cirrus CI](https://cirrus-ci.org)                                              | `ci.CIRRUS`             | ✅   |
+| [Codefresh](https://codefresh.io/)                                              | `ci.CODEFRESH`          | ✅   |
+| [Codeship](https://codeship.com)                                                | `ci.CODESHIP`           | 🚫   |
+| [Drone](https://drone.io)                                                       | `ci.DRONE`              | ✅   |
+| [dsari](https://github.com/rfinnie/dsari)                                       | `ci.DSARI`              | 🚫   |
+| [Expo Application Services](https://expo.dev/eas)                               | `ci.EAS`                | 🚫   |
+| [Gerrit CI](https://www.gerritcodereview.com)                                   | `ci.GERRIT`             | 🚫   |
+| [GitHub Actions](https://github.com/features/actions/)                          | `ci.GITHUB_ACTIONS`     | ✅   |
+| [GitLab CI](https://about.gitlab.com/gitlab-ci/)                                | `ci.GITLAB`             | ✅   |
+| [GoCD](https://www.go.cd/)                                                      | `ci.GOCD`               | 🚫   |
+| [Google Cloud Build](https://cloud.google.com/build)                            | `ci.GOOGLE_CLOUD_BUILD` | 🚫   |
+| [Hudson](http://hudson-ci.org)                                                  | `ci.HUDSON`             | 🚫   |
```

Comment:
Heroku is missing from this list:
```suggestion
| [Heroku](https://www.heroku.com)                                           | `ci.HUDSON`             | 🚫   |
| [Hudson](http://hudson-ci.org)                                                  | `ci.HUDSON`             | 🚫   |
```

Reply:
Good catch, added this to the heroku commit